### DISCUSSION TITLE
how to force proxy to use local cache?

### DISCUSSION THREAD

**Discussion by farwydi** [etcd-io/etcd#17823] [Q&A]
Hello
I'm not an etcd expert
I want to achieve a mechanism similar to replica in postgres from the etcd cluster

There are two data centers dc1 and dc2
communication delay between dc1 and dc2 is on average 100ms

etcd-3.5.13 version
main cluster etcd raise to dc1
in dc2 etcd grpc-proxy is raised which is aimed at the main cluster (in dc1)
in dc2 only readers from etcd grpc-proxy

when accessing etcd grpc-proxy (in dc2) I expect to see a minimum latency of less than 100ms
But in reality the delay increases to 100ms
As far as I understand, etcd grpc-proxy redirects the request to the leader of the main cluster because of this and there is a delay

The question is: is it possible to obtain a copy of the data in dc2 at the cost of the data being out of date?
And is this possible?

**Comment by farwydi** [comment]
https://github.com/etcd-io/etcd/issues/1330

here's something similar
but that was a long time ago

**Comment by jmhbnz** [comment]
Hey @farwydi - Thanks for your question, one potential option could be the mirror functionality of `etcdctl` https://github.com/etcd-io/etcd/blob/main/etcdctl/doc/mirror_maker.md
